Bottom: e366cf8d2cb8a5940db636f322475db4e4cbd09f
Top:    3a1dcfb81a4f659843abc458ee0d2b145a05eca3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-08 16:31:07 -0500

Refresh of add-a-demonstration-model-for-

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index d14b1da..97b2991 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1836,7 +1836,7 @@ via_hole_cyl_callback (const BoxType * b, void *cl)
 static int
 frontE_package_callback (const BoxType * b, void *cl)
 {
-  ElementTypePtr element = (ElementTypePtr) b;
+  ElementType *element = (ElementType *) b;
 
   if (FRONT (element))
     {
@@ -1852,7 +1852,7 @@ frontE_package_callback (const BoxType * b, void *cl)
 }
 
 static void
-ghid_draw_packages (BoxTypePtr drawn_area)
+ghid_draw_packages (BoxType *drawn_area)
 {
   /* XXX: Just the front elements for now */
   r_search (PCB->Data->element_tree, drawn_area, NULL, frontE_package_callback, NULL);
